nilearn/plotting/cm.py

```diff
@@ -157,7 +158,7 @@ def alpha_cmap(color, name='', alpha_min=0.5, alpha_max=1.):
 # Build colormaps and their reverse.
 _cmap_d = dict()
 
-for _cmapname in _cmaps_data.keys():
+for _cmapname in list(_cmaps_data.keys()):
```

Comment:
just wondering whether you need the list here, since you are just iterating over the keys